### DISCUSSION TITLE
M2A Fields Display Template

### DISCUSSION THREAD

**Discussion by st11x** [directus/directus#13760] [Archive]
Hi,

I have a M2A field from a collection type Navigation to Page. In the Navigation content form, I see the relations but they do not have any display values.

![image](https://user-images.githubusercontent.com/7398499/172239494-99b08448-daf1-4ee8-9385-9942a0b7cf28.png)

So I edited the field to make changes to the Display template, but nothing works.

![image](https://user-images.githubusercontent.com/7398499/172239596-fe553e89-0878-4022-b666-2f83a7daccb1.png)

However, if I made a change to the Page collection display template, then I could get something to show up. 

Why did the former not work?

I could not get fields that have been translated to show up. In this case, the Title is the translated field and it wouldn't show up. 

![image](https://user-images.githubusercontent.com/7398499/172239714-8e468869-f37c-4ccc-a318-231ae006de27.png)

In our use case, most of our fields have been moved into the translations as we need to support at least 2 languages. 

In doing so, it is breaking the display of fields in the CMS as Directus doesn't seem to be able to handle it.

thanks
  Matt

**Comment by rijkvanzanten** [comment]
The related values display / display template system doesn't currently support m2a scopes yet. Duplicate of https://github.com/directus/directus/issues/10601
